setup.py

```diff
@@ -33,7 +33,7 @@
     namespace_packages=["lookout"],
     keywords=["machine learning on source code", "babelfish", "lookout"],
     install_requires=[
-        "sourced-ml>=0.7.2,<0.8",
+        "sourced-ml>=0.8.0,<0.9",
         "lookout-sdk-ml>=0.9.0,<0.10",
```

Comment:
sdk-ml needs to be updated to 0.11

Reply:
The purpose of this PR was to alleviate #592 with some test fixes.
Updating to sdk-ml 0.11 Requires a lot of work Thzt is done in the other PR